scene/gui/scroll_container.cpp

```diff
@@ -37,10 +37,11 @@
 Size2 ScrollContainer::get_minimum_size() const {
 	Ref<StyleBox> sb = get_theme_stylebox(SNAME("bg"));
 	Size2 min_size;
+	Size2 size_max;
```

Comment:
This could be renamed to something like `content_min_size` for better clarity.

Reply:
Thanks for the suggestion.